makefiles/features_modules.inc.mk

```diff
@@ -24,11 +24,7 @@ PERIPH_IGNORE_MODULES := \
   periph_flash \
   periph_flashpage_in_address_space \
   periph_flexcomm \
-  periph_gpio_ll \
-  periph_gpio_ll_irq \
-  periph_gpio_ll_irq_level_triggered_high \
-  periph_gpio_ll_irq_level_triggered_low \
-  periph_gpio_ll_irq_unmask \
+  periph_gpio_ll% \
```

Comment:
I thought we wanted to get rid of wildcard pseudo-modules because those are impossible to find.

Reply:
Yes we do!

Here the issue was that the build system magically turns features into modules and magically adds init modules for the features starting with `periph_`.

The list of features added will still be checked against the list in `features.yaml`, so one cannot by accident add a feature and all features will be easily found and documented there.

IMO, we should expose the list of features used to C code (so that we do not need to use `if IS_USED(MODULE_PERIPH_GPIO_LL_IRQ_LEVEL_TRIGGERED_HIGH)) and only add modules with the same name of a feature if there is corresponding implementation code added. But that will be some work to implement ...

Reply:
In fact, if we would not have this wild-card pseudo module here:

https://github.com/RIOT-OS/RIOT/blob/37269c8ec74f3c4181bf144511e1ce7f5e349921/makefiles/pseudomodules.inc.mk#L639-L640

a warning would have been triggered about some `periph_init_gpio_ll_disconnect` module that does not exist.

Reply:
Was I able to convince you, or should I revert [this change](https://github.com/RIOT-OS/RIOT/compare/710fb905d7900f5f2ddce2b4028839dd35dc1bde..939a0488e99986021e6ade3868d432d71c2fd272)?